packages/ui-extensions/src/surfaces/point-of-sale/api/action-target-api/action-target-api.ts

```diff
@@ -1,7 +1,9 @@
 import {ScannerApi} from '../scanner-api/scanner-api';
 import {StandardApi} from '../standard/standard-api';
+import {CameraApi} from '../camera-api/camera-api';
 
 export type ActionTargetApi<T> = {[key: string]: any} & {
```

Comment:
Maybe I missed it but we only want to allow the camera to be triggered from action targets? Is there a reason why we don't want the api accessible from tiles or blocks?